### PR TITLE
new module: stock_ownership_by_move (see README)

### PR DESCRIPTION
**Pull request by lepistone**

### CODE REVIEW

File: stock_ownership_by_move/README.rst [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+Installing this module, the setting to show owners will be enabled
+automatically.
+
+Usage
+=====
+
+- In a picking, you can set the owner on every line.
+- When transferring a picking, lines with different owners will not be grouped.
+- In the transfer details wizard, the owner of every line can be changed.
+- After the trasfer, the owner is also shown in the Operations tab.
```
Comment (yvaucher) [inline stock_ownership_by_move/README.rst]: s/trasfer/transfer

File: stock_ownership_by_move/view/picking.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+<openerp>
+  <data>
+
+    <record id="view_picking_form" model="ir.ui.view">
+      <field name="name">view_picking_form</field>
+      <field name="model">stock.picking</field>
+      <field name="inherit_id" ref="stock.view_picking_form"/>
+      <field name="arch" type="xml">
+
+        <xpath expr="//field[@name='owner_id']/.." position="replace"/>
```
Comment (yvaucher) [inline stock_ownership_by_move/view/picking.xml:11]: Might be better to hide it instead of removing it. This would ease inheritance.

**Comment by yvaucher** [comment]
Only few remarks to fix

**Comment by guewen** [comment]
LGTM apart the things raised by @yvaucher

File: stock_ownership_by_move/view/picking.xml (hunk 2 above)
Comment (lepistone) [inline stock_ownership_by_move/view/picking.xml:11]: It is a div, containing a label, a field and a button. I don't think I can hide it.

**Comment by lepistone** [comment]
@yvaucher fixed and tests added.

File: stock_ownership_by_move/view/picking.xml (hunk 2 above)
Comment (yvaucher) [inline stock_ownership_by_move/view/picking.xml:11]: https://github.com/OCA/OCB/blob/8.0/addons/account_voucher/voucher_payment_receipt_view.xml#L535

It seems possible. There are multiple div tag using attrs to set invisible

**Comment by yvaucher** [comment]
@lepistone I answered your comment about the view part annihilation.

**Comment by jgrandguillaume** [comment]
Tested: works perfectly. :+1:

File: stock_ownership_by_move/view/picking.xml (hunk 2 above)
Comment (lepistone) [inline stock_ownership_by_move/view/picking.xml:11]: @yvaucher I tried this https://github.com/lepistone/stock-logistics-workflow/commit/af97a90163155fb4e75959f1db1a98c8da47c269 (with `eval`) but it doesn't work (I still see the owner).

Comment (yvaucher) [inline stock_ownership_by_move/view/picking.xml:11]: Ok then let's keep it that way

**Comment by yvaucher** [comment]
:+1:

**Pull request merged**